cpp/src/reductions/scan/scan_inclusive.cu

```diff
@@ -182,7 +182,8 @@ std::unique_ptr<column> scan_inclusive(column_view const& input,
 
   auto output = scan_agg_dispatch<scan_dispatcher>(
     input, agg, static_cast<bitmask_type*>(mask.data()), stream, mr);
-  output->set_null_mask(std::move(mask), null_count);
+  // Use the null mask produced by the op for EWM
+  if (agg.kind != aggregation::EWMA) { output->set_null_mask(std::move(mask), null_count); }
```

Comment:
I'm OK with this, but open question for other reviewers if they would prefer a more systematic approach to handling this. I'm inclined to leave it as-is for now and not try to overgeneralize to a single case.